frontend/projects/upgrade/src/app/features/dashboard/segments-legacy/components/modal/import-segment/import-segment.component.ts

```diff
@@ -45,7 +45,7 @@ export class ImportSegmentComponent {
         this.segmentDataService.importSegments(this.fileData)
       )) as importError[];
       this.showNotification(importResult);
-      this.segmentsService.fetchAllSegments(true);
+      this.segmentsService.fetchAllSegmentListOptions();
     } catch (error) {
       console.error('Error during segment import:', error);
     }
```

Comment:
Just curious, why do we need to fetch segments when the "Import Segment" button is clicked?

Reply:
removed, not sure why it used to do fetchAllSegments either, but it works fine without any fetch